drivers/netty/pom.xml

```diff
@@ -35,7 +35,7 @@
     <jclouds.osgi.export>org.jclouds.netty*;version="${project.version}"</jclouds.osgi.export>
     <jclouds.osgi.import>
       org.jclouds*;version="${project.version}",
-      org.jboss.netty*;version="[3.3,4)",
+      io.netty*;version="[3.3,4)",
```

Comment:
Why this change? AFAIK with the [version of netty we use](https://github.com/jclouds/jclouds/blob/master/drivers/netty/pom.xml#L57-L61) the packages we need are already [org.jboss.netty*](https://github.com/jclouds/jclouds/blob/master/drivers/netty/src/main/java/org/jclouds/netty/io/ChunkedFileInputStream.java#L24-L25).